web/shared-beans/src/test/java/io/deephaven/web/shared/data/RangeSetTest.java

```diff
@@ -408,8 +408,48 @@ public void testRemove() {
         assertEquals(of(
                 new Range(5, 10),
                 new Range(15, 20)), rangeSet);
+    }
+
+    @Test
+    public void testLarge() {
+        long largeA = 16535734029L;
+        long largeB = 36535734029L;
```

Comment:
ditto these constants, just to be sure we know what they are.

Maybe consider a reallyLargeA/etc, for Long.MAX_VALUE - 100 or something, though that will break when we pass it to JS